setup.py

```diff
@@ -63,8 +63,9 @@
         'seaborn',
         'yapf==0.29.0',
         'responses~=0.12.1',
+        'MarkupSafe==2.0.1', 
```

Comment:
You should upgrade your branch with main branch of official repo for each PR, these modification is from your last PR. (You can ignore this this time, but it is necessary to care about it in future work)